File: PJDesignBackStageClient/src/app/features/portfolio/feature-shared/models/get-portfolios.ts
import { EditResponseBase } from "src/app/shared/models/bases";
import { Category } from "src/app/shared/models/category";

export class GetPortfoliosResponse extends EditResponseBase {
  id: number;
  isBefore: boolean;
  title: string;
  categories: Category[];
  createDt: Date;
  isEnabled: boolean;
  date?: Date;
}

File: PJDesignBackStageAPI/App.Model/Portfolio/GetPortfolios.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class GetPortfoliosResponse : EditResponseBase
    {
        public int Id { get; set; }
        public bool IsBefore { get; set; }
        public string Title { get; set; } = null!;
        public DateTime? Date { get; set; }
        public DateTime CreateDt { get; set; }
        public bool IsEnabled { get; set; }
        public IEnumerable<Category>? Categories { get; set; }
    }
}
